Kernel/Thread.h

```diff
@@ -1241,7 +1239,7 @@ class Thread
     u32 m_kernel_stack_top { 0 };
     OwnPtr<Region> m_kernel_stack_region;
     VirtualAddress m_thread_specific_data;
-    SignalActionData m_signal_action_data[32];
+    SignalActionData m_signal_action_data[32] { };
```

Comment:
Suggest making this an `Array<SignalActionData, 32>` instead and adding `Array::fill()`

Reply:
`Array<SignalActionData, NSIG>` even :)